src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1906,32 +1906,44 @@ def list_av_sets(cmd, resource_group_name=None):
 
 # region VirtualMachines BootDiagnostics
 def disable_boot_diagnostics(cmd, resource_group_name, vm_name):
-    vm = get_vm_to_update(cmd, resource_group_name, vm_name)
-    diag_profile = vm.diagnostics_profile
-    if not (diag_profile and diag_profile.boot_diagnostics and diag_profile.boot_diagnostics.enabled):
+    from .aaz.latest.vm import Update as UpdateVM, Show as GetVM
```

Comment:
It is better to import vm show from which is under operations folder, since the output needs to be processed:
https://github.com/Azure/azure-cli/blob/2690e056ec67d59d7a8af88e6ba1706769ffe7c4/src/azure-cli/azure/cli/command_modules/vm/operations/vm.py#L13